Jenkinsfile

```diff
@@ -51,7 +51,7 @@ pipeline {
             sh 'git submodule sync --recursive'
             sh 'git submodule update --init --recursive --force'
             sh 'ccache -z'
-            sh 'cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_BACKEND=ON -DBUILD_SHARED_LIBS=OFF -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX=${WORKSPACE}/install -DENABLE_MAVLINK_PASSTHROUGH=ON -Bbuild/release -H.'
```

Comment:
How is that related to Jenkins crashes? And to the cmake cache?

Reply:
No that's not responsible, sorry.

Reply:
So I made it this way (i.e. static libs) in Jenkins because Travis does the shared libs: https://github.com/mavlink/MAVSDK/blob/develop/tools/travis-docker-build.sh#L8.

Do you think we should just build shared libs everywhere in CI?

Reply:
I didn't change the function here, I only removed the duplicate flag.

Reply:
Oh, I just understood it... damn it took me long enough to realize what you meant :sweat_smile: